src/lib/CONFIG/config.h

```diff
@@ -52,7 +52,8 @@ typedef struct {
                 txAntenna:2,    // FUTURE: Which TX antenna to use, 0=Auto
                 ptrStartChannel:4,
                 ptrEnableChannel:5,
-                _unused:3;
+                linkMode:2,
+                _unused:1;
```

Comment:
I would future proof this a little and use the last spare bit.
`
linkMode:3,`

Reply:
I considered doing that while I was there, but it felt selfish stealing bits that I don't need just yet. Happy to change it to 3-bit if we foresee more getting added here in future...

Reply:
If we are becoming the new mulit-protocol module... we'll need the bits.